src/access/AccessControl.cpp

```diff
@@ -193,22 +193,107 @@ CHIP_ERROR AccessControl::Finish()
     return retval;
 }
 
-CHIP_ERROR AccessControl::RemoveFabric(FabricIndex fabricIndex)
+CHIP_ERROR AccessControl::CreateEntry(const SubjectDescriptor * subjectDescriptor, FabricIndex fabric, size_t * index,
+                                      const Entry & entry)
 {
-    ChipLogProgress(DataManagement, "AccessControl: removing fabric %u", fabricIndex);
+    VerifyOrReturnError(IsInitialized(), CHIP_ERROR_INCORRECT_STATE);
 
-    CHIP_ERROR err;
-    do
+    size_t count    = 0;
+    size_t maxCount = 0;
+    ReturnErrorOnFailure(mDelegate->GetEntryCount(fabric, count));
+    ReturnErrorOnFailure(mDelegate->GetMaxEntriesPerFabric(maxCount));
```

Comment:
Could this just call our `GetMaxEntriesPerFabric` method?

Reply:
I think it's OK to just call the delegate.